lambda-notes/src/App.js

```diff
@@ -0,0 +1,89 @@
+import React, { Component } from 'react';
+import './App.css';
+import { Route, withRouter } from 'react-router-dom';
+import axios from 'axios';
+
+
+import LeftBar from './components/LeftBar';
+import NotesList from './components/NotesList';
+import AddNoteForm from './components/AddNoteForm';
+import ExpandedNote from './components/ExpandedNote';
+import EditNoteForm from './components/EditNoteForm';
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      notes: [],
+      expandedNote: {}
+    }
+  }
+
+  componentDidMount() {
+    axios
+      .get('https://fe-notes.herokuapp.com/note/get/all')
+      .then(response => this.setState({ notes: response.data.reverse() }))
+      .catch(error => console.log(error));
+  }
+
+  addNote = note => {
+    console.log('hello');
+    axios
+        .post('https://fe-notes.herokuapp.com/note/create', note)
+        .then(() => {
+          let newNotes = [...this.state.notes];
+          newNotes.unshift(note);
```

Comment:
this works great! simpler would be to just do: `let newNotes = [...this.state.notes, note];`